isisd/isis_sr.c

```diff
@@ -1781,6 +2071,27 @@ DEFUN(show_sr_node, show_sr_node_cmd,
 
 /* --- IS-IS Segment Routing Management function ---------------------------- */
 
+/**
+ * Thread function to re-attempt connection to the Label Manager and thus be
+ * able to start Segment Routing.
+ *
+ * @param start		Thread structure that contains area as argument
+ *
+ * @return		1 on success
+ */
+static int sr_start_label_manager(struct thread *start)
+{
+	struct isis_area *area;
+
+	area = THREAD_ARG(start);
+	area->srdb.t_start_lm = NULL;
```

Comment:
you can't touch this variable here; it's managed under by the libthread / thread master library, under a mutex.

Reply:
OK. I'll remove that.

Reply:
Done